cocos/asset/asset-manager/deprecated.ts

```diff
@@ -532,7 +532,9 @@ export class CCLoader {
      *
      * @deprecated since v3.0 loader.md5Pipe is deprecated, assetLoader and md5Pipe were merged into assetManager.transformPipeline
      */
-    public get md5Pipe () {
+    public get md5Pipe (): {
+        transformURL(url: string): string;
```

Comment:
The type seems strange.

Reply:
that is because this getter returns an object `md5Pipe`, and `md5Pipe` is an untyped js object, 

so only an automatically inferred type can be generated here